### PR TITLE
various readme updates

### PR DESCRIPTION
**Pull request by microwavedcola1**
Just documenting how we release, adding some other placeholder documents, please suggest whatever you can think of, we can fill them out later, useful to get outline in imo

Signed-off-by: microwavedcola1 <microwavedcola@gmail.com>

### CODE REVIEW

**Comment by microwavedcola1** [comment]
For better reading, use 
image

File: README.md [hunk 1]
```diff
@@ -16,18 +20,23 @@ Consider setting the git option `submodule.recurse=true`.
...
+
+### Release
+
+Here are steps followed while performing a program deployment to mainnet-beta
 
-### Notes
+- review diff of last deployed tag to mainnet-beta, e.g. https://github.com/blockworks-foundation/mango-v4/compare/program-v0.4.0..dev, pay special attention to account layout changes, backward compatibility of newly introduced account fields, etc.
+- update changelog
+- add a git tag e.g. `program-v0.0.1`, should match the version the program has
+- deploy to mainnet-beta
```
Comment (ckamm) [inline README.md:32]: typically I deploy first, then put the deploy timestamp/tx info into the changelog and only then tag the version

File: README.md [hunk 2]
```diff
@@ -16,18 +20,23 @@ Consider setting the git option `submodule.recurse=true`.
...
+- merge code to `main` branch
```
Comment (ckamm) [inline README.md:33]: never merge to main, just reset main to the currently deployed tag

**Review by ckamm — COMMENTED** [review]
This has changes to the anchor submodule! Probably unintentional?

**Pull request merged**